backend/app/src/avatar/avatar.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, Post, UseInterceptors, Bind, UploadedFile, UseGuards, Get, Res, Param } from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { JwtFtGuard } from 'auth/ft/jwt-ft.strategy';
+import { multerOptions } from './avatar.options';
+import { Response } from 'express';
+
+@Controller('api/avatar')
+export class AvatarController {
+
+	@Post()
+	@UseInterceptors(FileInterceptor('file', multerOptions('/srv/uploads/avatar')))
+	@Bind(UploadedFile())
+	async upload(@UploadedFile() file) {
+		return file.filename;
+	}
+
+	@Get(':uuid')
+	async get(@Res() res : Response, @Param('uuid') uuid : string) {
+		res.download(`/srv/uploads/avatar/${uuid}`, uuid);
+	}
+
+}
```

Comment:
```suggestion
@Controller('api/avatar')
@ApiTags('유저 API')
export class AvatarController {

	@Post()
	@ApiOperation({ summary: '아바타 저장', description: 'DB에 아바타 이미지 파일을 저장' })
	@ApiCreatedResponse({ description: '저장 성공', type: string }) // 이미지 타입?
	@UseInterceptors(FileInterceptor('file', multerOptions('/srv/uploads/avatar')))
	@Bind(UploadedFile())
	async upload(@UploadedFile() file) {
		return file.filename;
	}

	@Get(':uuid')
	@ApiOperation({ summary: '아바타 불러오기', description: 'uuid로 아바타 이미지 파일 불러오기' })
	@ApiCreatedResponse({ description: '불러기 성공', type: string })
	async get(@Res() res : Response, @Param('uuid') uuid : string) {
		res.download(`/srv/uploads/avatar/${uuid}`, uuid);
	}

}
```
가령 이런 식으로 swagger 문서화도 추가해주시면 좋을 것 같습니다.